src/aem/azext_aem/custom.py

```diff
@@ -0,0 +1,546 @@
+            sla = EnhancedMonitoring._get_disk_sla(os_disk['size'])
+            pub_cfg.update({
+                'osdisk.type': 'Premium',
+                'osdisk.sla.throughput': sla['TP'],
+                'osdisk.sla.iops': sla['IOPS'],
+            })
+        if managed_disk and not os_disk['is_premium']:
+            logger.warning('Standard Managed Disks are not supported. '
+                           'Extension will be installed but no disk metrics will be available.')
+        else:
+            pub_cfg.update({
+                'osdisk.name': os_disk['name'],
+                'osdisk.caching': os_disk['caching'],
+            })
+            if not managed_disk:
+                pub_cfg.update({
+                    'osdisk.account': os_disk['account_name']
+                })
+            if not os_disk['is_premium']:
+                pub_cfg.update({
+                    'osdisk.type': 'Standard',
+                    'osdisk.connminute': os_disk['account_name'] + '.minute',
+                    'osdisk.connhour': os_disk['account_name'] + '.hour',
+                })
+        # data disks
+        for i, disk in enumerate(disk_info['data_disks']):
+            suffix = '.' + str(i + 1)
+            if disk['is_premium']:
+                sla = EnhancedMonitoring._get_disk_sla(disk['size'])
+                pub_cfg.update({
+                    'disk.type' + suffix: 'Premium',
+                    'disk.sla.throughput' + suffix: sla['TP'],
+                    'disk.sla.iops' + suffix: sla['IOPS'],
+                })
+
+            if managed_disk and not disk['is_premium']:
+                logger.warning('Standard Managed Disks are not supported. '
+                               'Extension will be installed but no disk metrics will be available.')
+            else:
+                pub_cfg.update({
+                    'disk.lun' + suffix: disk['lun'],
+                    'disk.name' + suffix: disk['name'],
+                    'disk.caching' + suffix: disk['caching'],
+                })
+                if not managed_disk:
+                    pub_cfg.update({
+                        'disk.account' + suffix: disk['account_name']
+                    })
+                if not os_disk['is_premium']:
+                    pub_cfg.update({
+                        'disk.type' + suffix: 'Standard',
+                        'disk.connminute' + suffix: disk['account_name'] + '.minute',
+                        'disk.connhour' + suffix: disk['account_name'] + '.hour',
+                    })
+
+        if not managed_disk:
+            unmanaged_disks = [disk_info['os_disk']] + disk_info['data_disks']
+            for disk in unmanaged_disks:
+                account_name = disk['account_name']
+                if disk['is_premium']:
+                    logger.info('[INFO] %s is skipped - Storage Account Metrics are not available '
+                                'for Premium Type Storage.', disk['name'])
```

Comment:
Why the `[INFO]` prefix here?
The logger.info would handle whether to add that prefix or not.